src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -39,7 +40,7 @@ public class DefaultContentManager : IContentManager
             ReversedHandlers = handlers.Reverse().ToArray();
             _session = session;
             _idGenerator = idGenerator;
-            _contentManagerSession = new DefaultContentManagerSession();
```

Comment:
@jtkech @sebastienros is this for a reason?

Reply:
I don't remember exactly

Maybe just because it's a standalone class that doesn't inject any services and here the instance is held by the scoped `DefaultContentManager` that is the only one that uses it.

Maybe we could just let it as it is and then remove the scoped registration of `IContentManagerSession` and maybe delete the defenition of this interface.

Reply:
@sebastienros can we take action on this one, while breaking-changes is allowed?